modules/network-observability-ebpf-rule-flow-filter.adoc

```diff
@@ -0,0 +1,16 @@
+// Module included in the following assemblies:
+//
+// network_observability/observing-network-traffic.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="network-observability-ebpf-flow-rule-filter_{context}"]
+= eBPF flow rule filter
+You can use rule-based filtering to refine where the eBPF agent deploys and what is collected. The rule-based filtering is helpful for controlling the volume of packets cached in the eBPF flow table. For example, a filter can specify that only packets coming from port 100 should be recorded. Then all packets which do not match the filter are dropped and only the ones that match pass.
```

Comment:
well to decide where the agent will be deployed we have nodeSelector/tolartions etc which is irrelevant to flow filtering

Reply:
packets that don't match filters aren't dropped they are just not cached in netobserv flow table so no longer visible from NOO pov

Reply:
> well to decide where the agent will be deployed we have nodeSelector/tolartions etc which is irrelevant to flow filtering

OK, maybe I misinterpreted what you wrote in the Jira [epic](https://issues.redhat.com/browse/NETOBSERV-1598) for this: 
# Epic Goal
This epic will help customers to fine grain where eBPF agents deploys and what is collected.

I will remove this detail. 

> packets that don't match filters aren't dropped they are just not cached in netobserv flow table so no longer visible from NOO pov

Okay, thanks I will update this detail.